kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/api/KuronometerApiClientConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.api
+
+case class KuronometerApiClientConfig(
+                                       scheme: String = "https",
```

Comment:
I can't use this formatter because the build tool is  not sbtGradle :S I tired it but as far as I know it's not supported